packages/mui-material/src/Select/Select.d.ts

```diff
@@ -151,23 +151,26 @@ export interface BaseSelectProps<Value = unknown>
   variant?: SelectVariants;
 }
 
-export interface FilledSelectProps extends Omit<FilledInputProps, 'value' | 'onChange'> {
+export interface FilledSelectProps
+  extends Omit<FilledInputProps, 'value' | 'onChange' | 'id' | 'classes' | 'inputProps'> {
```

Comment:
We need to omit these props in the types because the `pnpm proptypes` [started to fail](https://app.circleci.com/pipelines/github/mui/material-ui/126317/workflows/c2bf08ed-4a3e-47b2-b554-0897d84265ef/jobs/681347) after the change in the Select definition below. I believe this adjustment is now correct. Previously, it may not have been picking up the props correctly, possibly due to an incorrect intersection with `muiName: string`.